services/wallet/bridge/cbridge.go

```diff
@@ -60,43 +63,27 @@ func (s *CBridge) Name() string {
 }
 
 func (s *CBridge) estimateAmt(from, to *params.Network, amountIn *big.Int, symbol string) (*cbridge.EstimateAmtResponse, error) {
-	client := &http.Client{
-		Timeout: time.Second * 5,
-	}
 	base := baseURL
 	if from.IsTest {
 		base = testBaseURL
 	}
-	url := fmt.Sprintf(
-		"%s/v2/estimateAmt?src_chain_id=%d&dst_chain_id=%d&token_symbol=%s&amt=%s&usr_addr=0xaa47c83316edc05cf9ff7136296b026c5de7eccd&slippage_tolerance=500",
-		base,
-		from.ChainID,
-		to.ChainID,
-		symbol,
-		amountIn.String(),
-	)
 
-	req, err := http.NewRequest(http.MethodGet, url, nil)
-	if err != nil {
-		return nil, err
-	}
+	params := netUrl.Values{}
+	params.Add("src_chain_id", strconv.Itoa(int(from.ChainID)))
+	params.Add("dst_chain_id", strconv.Itoa(int(to.ChainID)))
+	params.Add("token_symbol", symbol)
+	params.Add("amt", amountIn.String())
+	params.Add("usr_addr", "0xaa47c83316edc05cf9ff7136296b026c5de7eccd")
+	params.Add("slippage_tolerance", "500")
 
-	resp, err := client.Do(req)
+	url := fmt.Sprintf("%s/v2/estimateAmt", base)
+	response, err := s.httpClient.DoGetRequest(context.Background(), url, params)
```

Comment:
Were you avoiding vars shadowing the `url` import with the `netUrl` alias? If yes, the convention (but not the rule) is to change the variable name, rather than the import alias.

Shadowing in Go is allowed, but many devs (myself included) avoid it like you have.

My opinion is that It is fine to make no changes here, just to point out that import aliasing is normally done to handle multiple package imports with the same name, and communicates that the dev should be careful when choosing which package they are selecting.